addons/mail/static/src/core/common/composer.xml

```diff
@@ -33,7 +33,7 @@
             </div>
             <div class="o-mail-Composer-coreHeader text-truncate small p-2" t-if="props.composer.thread and props.messageToReplyTo?.thread?.eq(props.composer.thread)">
                 <span class="cursor-pointer" t-on-click="() => env.messageHighlight?.highlightMessage(props.messageToReplyTo.message, props.composer.thread)">
-                    Replying to <b t-esc="props.messageToReplyTo.message.author?.name ?? props.messageToReplyTo.message.email_from"/>
+                    Replying to <b t-esc="props.messageToReplyTo.message.authorName ?? props.messageToReplyTo.message.email_from"/>
```

Comment:
Can remove `props.messageToReplyTo.message.email_from` part, this is handled in `authorName`.